frontend/lib/time/index.ts

```diff
@@ -0,0 +1,57 @@
+const defaultLocale = 'en-US'
+
+const getLocale = (): string => {
+  const { locale } = useI18n()
```

Comment:
It's generally not a good idea to call composables from anywhere except

1) other composables
2) plugins (and only if you're really careful and know what you're doing
3) setup functions

The risk of course being that this code gets called in an non-setup function, which is doubly likely because this function is then used by a bunch of functions below.

The best solution is likely to just move all of this into it's own `useLocale` or `useTime` composable, which calls `useI18n` at the top level and then returns all these other functions.